src/main/java/org/myrobotlab/service/Runtime.java

```diff
@@ -345,7 +449,7 @@ public final static void createAndStartServices(List<String> services) {
 
         log.info("attempting to invoke : {} of type {}", name, type);
 
-        ServiceInterface s = Runtime.create(name, type);
+        ServiceInterface s = Runtime.create(null, name, type);

```

Comment:
confusing to pass a null plan.. omit the null in this call and rely on the String name, String type definition instead.

Reply:
yeah - its going away